packaging/linux_simple/package.appimage.sh.ungoogin

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -eux
+
+CURRENTDIR=$(dirname $(readlink -f $0))
+
+pushd "$CURRENTDIR"
+
+if [ ! -d "./AppImages" ]; then
+  git clone https://github.com/AppImage/AppImages.git
```

Comment:
If `pkg2appimage` is a standalone script, it may be better just to retrieve it directly via HTTPS.

Otherwise, I think it would be better to use a shallow clone here to save a little bandwidth.